cmd/archeio/app/handlers.go

```diff
@@ -26,6 +26,11 @@ import (
 	"sigs.k8s.io/oci-proxy/pkg/net/cidrs/aws"
 )
 
+const (
+	// hard coded for now
+	infoURL = "https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io"
```

Comment:
I think we should probably switch to https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-(registry.k8s.io)

this link doesn't seem to have much relevant, we're not using the things that are there

Reply:
will try to file a follow up soon. this just came up after the community update today